src/com/company/Sudoku.java

```diff
@@ -0,0 +1,380 @@
+package com.company;
+
+import javax.swing.*;
+import java.util.Arrays;
+import java.util.Scanner;
+import java.util.regex.Pattern;
+
+public class Sudoku {
+    public Sudoku(Scanner gameScan) {
+        this.gameScan = gameScan;
+    }
+
+    private  final Scanner gameScan;
+
+    public  boolean startup() {
+
+        System.out.println("Witamy w Sudoku Zugaj Edition V1\nBy rozpocząć rozgrywkę wpisz:  Sudoku");
+        String x= gameScan.next();
+        return x.equals("Sudoku");
+    }
+
+    public boolean validatePion(int[][] board){
```

Comment:
validate pion 🤢
Używaj pełnego angielskiego oki
I jakbyś miał int[][] board jako pole w klasie to nie musiałbyś teraz go tu mieć w parametrach. Kozak
Ogólnie jeśli metoda nie ma żadnych parametrów to jest idealna i powinno się do tego dążyć